tools/section-link-check.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+for file in `find . -name "*.md" -type f`; do
+  echo "Checking $file..."
+  sections=$(pandoc $file --verbose --fail-if-warnings -o pdfs/tmp.pdf 2>&1 | grep -E 'pdfTeX warning \(dest\): name{[^}]+}' | sed -E  's/.*name\{([^}]+)\}.*/\1/' | sort | uniq)
+  if [[ $(wc -l <<< "$sections") -gt 0 ]]; then
+    echo "**** WARNING! These section links have been found to not be working: "
+    echo $sections | tr " " "\n"
```

Comment:
Something fishy is going on here because we don't see the list of broken links: https://github.com/ARM-software/acle/runs/4169294120?check_suite_focus=true#step:8:8